packages/react-log-viewer/src/LogViewer/loggerToolbar.tsx

```diff
@@ -0,0 +1,177 @@
+import React, { useState, useEffect } from 'react';
+import { css } from '@patternfly/react-styles';
+import DownloadIcon from '@patternfly/react-icons/dist/js/icons/download-icon';
+import ExpandIcon from '@patternfly/react-icons/dist/js/icons/expand-icon';
+import ExternalLinkAltIcon from '@patternfly/react-icons/dist/js/icons/external-link-alt-icon';
+import PlayIcon from '@patternfly/react-icons/dist/js/icons/play-icon';
+import { useLoggerContext } from './LoggerContext';
+import ToolbarDropdown from './toolbarDropdown';
+import { NUMBER_INDEX_DELTA } from './utils/constants';
+import { Button, SearchInput, Toolbar, ToolbarItem, ToolbarContent, ToolbarGroup } from '@patternfly/react-core';
+import styles from '@patternfly/react-styles/css/components/LogViewer/loggerToolbar.styles';
+
+export interface LoggerToolbarProps extends React.HTMLProps<HTMLDivElement> {
+  includesPlay?: boolean;
+  includesFullScreen?: boolean;
+  includesLaunchExternal?: boolean;
+  includesDownload?: boolean;
+  dataSourcesAmount?: number;
+  dataSourceTitles?: string[] | null[];
+  customToolbarActions?: () => React.ReactNode | React.ReactNode[];
+  scrollToRow: (searchedRow: number) => void;
+}
+
+const LoggerToolbar: React.FC<LoggerToolbarProps> = ({
+  scrollToRow,
+  // customToolbarActions,
```

Comment:
you are not rendering any customToolbarActions? It's not necessary for this PR anyway, so we can remove this prop and any references to it in other components for now I think its okay if the only item in the toolbar with this PR is the search input